spec.md

```diff
@@ -209,9 +154,9 @@ The V2 registry API does not enforce this.
 The rules for a repository name are as follows:
 
 1. A repository name is broken up into _path components_.
-   A component of a repository name must be at least one lowercase, alpha-numeric characters, optionally separated by periods, dashes or underscores.
-   More strictly, it must match the regular expression `[a-z0-9]+(?:[._-][a-z0-9]+)*`.
-2. If a repository  name has two or more path components, they must be separated by a forward slash ("/").
+A component of a repository name must be at least one of lowercase, alpha-numeric characters, optionally separated by periods, dashes or underscores.
```

Comment:
The original is unclear, but this change doesn't parse for me. What do you think about this?

`A component of a repository name must be comprised of lowercase, alpha-numeric characters (at least one), optionally separated by periods, dashes or underscores.`

Reply:
I think it must begin with `[[:alnum:]]`, but can also include `[.-_]`

Reply:
going with: "A component of a repository name must begin with one or more lowercase alpha-numeric characters. Subsequent lowercase alpha-numeric characters are optional and may be separated by periods, dashes or underscores."